app/scrapers/gigablast.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class Gigablast(Scraper):
+    """Scrapper class for Gigablast"""
+
+    def __init__(self):
+        self.url = 'https://www.gigablast.com/search'
+        self.defaultStart = 0
+        self.startKey = 'start_index'
+
+    def parseResponse(self, soup):
+        """ Parse the response and return set of urls
+        Returns: urls (list)
+                [[Tile1,url1], [Title2, url2],..]
+        """
+        urls = []
+        for a in soup.findAll('a', {'class': 'title'}):
```

Comment:
Gigablast also has description, it'd be good if we include that too.

Reply:
i didnt get you. can you please explain?

Reply:
Currently, this implementation sends title and a link, how about we include the description, summary provided as text from the search resuls ?

Reply:
https://github.com/fossasia/query-server/blob/master/app/scrapers/bing.py#L22 is an example of a scraper that has a description field.  Some search engines have them and others to do not.